### PR TITLE
[docs] Deckhouse Registry mirroring docs

### PR DESCRIPTION
**Pull request by mvasl**
## Description

Update FAQ: add a guide for mirroring the Deckhouse registry using the `dhctl mirror` tool.

## Changelog entries

```changes
section: docs
type: chore
summary: Add a guide for mirroring the Deckhouse registry using the `dhctl mirror` tool.
impact_level: default
```

### CODE REVIEW

File: docs/documentation/pages/DECKHOUSE-FAQ.md [hunk 1]
```diff
@@ -381,57 +381,59 @@ You need to use the Proxy Cache feature of a [Harbor](https://github.com/goharbo
...
+
+1. After pushing images to an isolated private registry, you are ready to install Deckhouse from it. Use the [Getting started](/gs/bm-private/step2.html) guide.
+
+   When starting the installer, use its image from the registry, into which Deckhouse images were previously pushed, and not from the public registry. For example, use an address like `your.private.registry.com:5000/deckhouse/ee/install:stable` instead of `registry.deckhouse.io/deckhouse/ee/install:stable`.
 
-1. After pushing images to an isolated private registry, use [the instruction](deckhouse-faq.html#how-to-bootstrap-a-cluster-and-run-deckhouse-without-the-usage-of-release-channels) to properly configure the installer and the `InitConfiguration` resource.
+   During installation add the address of your registry and authorization data to the `InitConfiguration` resource (parameters [imagesRepo](/documentation/v1/installing/configuration.html#initconfiguration-deckhouse-imagesrepo), [registryDockerCfg](/documentation/v1/installing/configuration.html#initconfiguration-deckhouse-registrydockercfg) or [step 3](/gs/bm-private/step3.html) of the Getting started guide).
```
Comment (kirkonru) [inline docs/documentation/pages/DECKHOUSE-FAQ.md:438]:
```suggestion
   During installation, add your registry address and authorization data to the `InitConfiguration` resource (the [imagesRepo](/documentation/v1/installing/configuration.html#initconfiguration-deckhouse-imagesrepo) and [registryDockerCfg](/documentation/v1/installing/configuration.html#initconfiguration-deckhouse-registrydockercfg) parameters; you might refer to [step 3](/gs/bm-private/step3.html) of the Getting started guide as well).
```

File: docs/documentation/pages/DECKHOUSE-FAQ.md [hunk 2]
```diff
@@ -471,7 +473,7 @@ To switch the Deckhouse cluster to using a third-party registry, follow these st
 
 ### How to bootstrap a cluster and run Deckhouse without the usage of release channels?
 
-Use this method only valid if you don't have release channel images in your air-gapped registry.
+Use of this method only valid if you don't have release channel images in your air-gapped registry.
```
Comment (kirkonru) [inline docs/documentation/pages/DECKHOUSE-FAQ.md:478]:
```suggestion
This method should only be used if there are no release channel images in your air-gapped registry.
```

**Review by kirkonru — APPROVED** [review]

**Review by z9r5 — APPROVED** [review]

File: docs/documentation/pages/DECKHOUSE-FAQ.md [hunk 3]
```diff
@@ -381,57 +381,61 @@ You need to use the Proxy Cache feature of a [Harbor](https://github.com/goharbo
...
 
-1. Download script on the host:
+   For example, here is how you can pull all Deckhouse version images starting from version 1.45:
 
    ```shell
-   curl -fsSL -o d8-push.sh https://raw.githubusercontent.com/deckhouse/deckhouse/main/tools/release/d8-push.sh
-   chmod 700 d8-push.sh
+   dhctl mirror --license="<DECKHOUSE_LICENSE_KEY>" --images-bundle-path /tmp/d8-images/d8.tar --min-version=1.45
    ```
+
```
Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE-FAQ.md:417]:
```suggestion

```

File: docs/documentation/pages/DECKHOUSE-FAQ.md [hunk 4]
```diff
@@ -381,57 +381,60 @@ You need to use the Proxy Cache feature of a [Harbor](https://github.com/goharbo
...
 
-1. Download script on the host:
+   For example, here is how you can pull all Deckhouse version images starting from version 1.45:
 
    ```shell
-   curl -fsSL -o d8-push.sh https://raw.githubusercontent.com/deckhouse/deckhouse/main/tools/release/d8-push.sh
-   chmod 700 d8-push.sh
+   dhctl mirror --license="<DECKHOUSE_LICENSE_KEY>" --images-bundle-path /tmp/d8-images/d8.tar --min-version=1.45
    ```
+   > Note that `--min-version` parameter will be ignored if you specify version above current rock-solid channel.
```
Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE-FAQ.md:417]:
```suggestion

   > Note that `--min-version` parameter will be ignored if you specify version above current rock-solid channel.
```

**Review by z9r5 — APPROVED** [review]

**Review by RomanenkoDenys — APPROVED** [review]
LGTM

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [6471](https://github.com/deckhouse/deckhouse/pull/6471) to the branch [release-1.54](https://github.com/deckhouse/deckhouse/tree/release-1.54) successful!

**Pull request merged**